cmd/integration_test/build/buildspecs/test-infra.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I will remove this when PR is ready and generate on the fly during the ci build. Just including it for now so reviewers can see what it will look like.